src/containers/TopbarContainer/Topbar/README.md

```diff
@@ -0,0 +1,102 @@
+
+The Topbar has built-in support for dynamic links based on data fetched from the
+[asset delivery API](https://www.sharetribe.com/api-reference/asset-delivery-api.html).
+
+The Topbar includes two primary components, one for the Desktop layouts and one for mobile layouts.
+The breakpoint is triggered at 1024px.
+
+## Links
+
+The Topbar supports two types of links: primary and secondary. Primary links are shown on the top
+level of the Topbar, i.e. they are always visible if the viewport is not below 1024px (which
+triggers the mobile layout). Secondary links are grouped under a "More" dropdown menu.
+
+Primary links will collapse under a dropdown menu if the viewport's width is not wide enough. Links
+are collapsed individually, i.e., the Topbar will always try to render as many primary links as
+possible. If the width of the Topbar is so narrows that no primary links fit, the dropdown menu is
+renamed from "More" to "Menu".
+
+The "Post a new listing" link is always shown as the first primary link, and it is prioritized in
+the visible space
+
+## CustomLinksMenu
+
+The CustomLinksMenu component dynamically manages and displays the custom links loaded through
+assets within the Topbar. It organizes links into two categories based on available space: priority
+links that are always visible in the Topbar, and secondary links grouped under a "More" dropdown
+menu. The component adjusts in real-time to browser resizing and page layout changes. It uses the
+ResizeObserver web API to monitor for changes in the viewport's width.
+
+This component is inherently somewhat fragile as it needs to deal with the DOM directly. If you
+customize the Topbar, make sure to test responsiveness thoroughly.
+
+## Hosted assets
+
+The dynamic links in the menu use a hosted asset called top-bar.json. An example of the structure:
+
+```
+{
+    "data": {
+      "customLinks": [
+        {
+          "group": "primary",
+          "type": "internal",
+          "href": "/p/about",
+          "text": "About page"
+        }
+      ],
+      "logoLink": {
+        "href": "http://www.example.com"
+      },
+      "id": "631q5e3q-13q5-4871-8dc0-923gg6be286b",
+      "type": "jsonAsset"
+    }
+}
+```
+
+## logoLink
+
+The logoLink property in the top-bar.json asset determines the destination URL of the logo. The
+logoLink property should only be used if you want your logo to link to an external URL. The
+intention here is to allow you to host your marketplace on a different domain from where you are
+hosting the rest of your site. For example, your site may be at https://your-website.com and your
+marketplace at https://marketplace.your-website.com. The logo (see LinkedLogo.js) automatically
+links to the landing page of your website if no logoLink property is present.
+
+Note: the logoLink property does not support in-app navigation at the moment. If the URL is the
+marketplace's domain (https://marketplace.famouswebsite.com), a full page refresh is performed every
```

Comment:
The previous example used domain: "https://marketplace.your-website.com/". So, either use that or something that explicitly says the domain is about marketplace (e.g. "https://your-marketplace.com")